Sources/Holocron/FileManager.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FileManager.swift
+//  Holocron-iOS
+//
+//  Created by Pranjal Satija on 4/29/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
I think I mentioned this before but we need to replace "Cocoapods" with "Bottle Rocket Studios" throughout the project.